plugins/org.jkiss.dbeaver.ext.altibase/src/org/jkiss/dbeaver/ext/altibase/model/AltibaseSQLDialect.java

```diff
@@ -0,0 +1,626 @@
+        return null;
+    }
+
+    @Override
+    public boolean supportsCreateIfExists() {
+        return false;
+    }
+
+    @Override
+    public String getTimestampDataType() {
+        return AltibaseDataTypeDomain.DATE.getTypeName();
+    }
+
+    @Override
+    public String getBigIntegerType() {
+        return AltibaseDataTypeDomain.BIGINT.getTypeName();
+    }
+
+    @Override
+    public String getBlobDataType() {
+        return AltibaseDataTypeDomain.BLOB.getTypeName();
+    }
+    
+    @Override
+    public String getClobDataType() {
+        return AltibaseDataTypeDomain.CLOB.getTypeName();
+    }
+
+    @Override
+    public String getUuidDataType() {
+        // Not supported data type
+        return "";
+    }
+
+    @Override
+    public String getBooleanDataType() {
+        // Not supported data type
+        return "";
+    }
+    
+    @Override
+    public String getSchemaExistQuery(String schemaName) {
+        return "SELECT 1 FROM SYSTEM_.SYS_USERS_ WHERE USER_NAME='" + schemaName + "'";
+    }
+
+    @Override
+    public String getCreateSchemaQuery(String schemaName) {
+        return "CREATE USER \"" + schemaName + "\" IDENTIFIED BY \"" + schemaName + "\"";
+    }
+    
+    /**
+     * Extracting and Formatting Altibase only keywords and functions 
+     * for ALTIBASE_ONLY_KEYWORDS and ALTIBASE_ONLY_FUNCTIONS.
+     */
+    public static void main(String[] args) {
+        filter(ALTIBASE_KEYWORDS, SQLConstants.SQL2003_RESERVED_KEYWORDS);
+        filter(ALTIBASE_FUNCTIONS, SQLConstants.SQL2003_FUNCTIONS);
+    }
+    
+    private static void filter(String[] dbProvideNames, String[] sqlProvidedNames) {
+        String[] sqlDistinctNames = null;
+        String[] dbDistinctNames = null;
+        
+        // Remove duplicates if any
+        sqlDistinctNames = Arrays.stream(sqlProvidedNames).distinct().toArray(String[]::new);
+        dbDistinctNames = Arrays.stream(dbProvideNames).distinct().toArray(String[]::new);
+        
+        // Find DB only words
+        Set<String> defaultWordList = Set.of(sqlDistinctNames);
+        final List<String> dbOnlyWordList = List.of(dbDistinctNames).stream()
+                .filter(e -> !defaultWordList.contains(e))
+                .sorted()
+                .collect(Collectors.toList());
+        
+        // Print out the result
+        int maxLen = dbOnlyWordList.stream()
+                .mapToInt(String::length)
+                .max().orElse(8);
+
+        maxLen++;
+        
+        int i = 0;
+        int numOfWordsLine = 2;
+        String wordFormat = "%-" + (maxLen + 2) + "s";
+        String numFormat = "\t//%3d";
+        
+        for (String dbOnlyWord : dbOnlyWordList) {
+            System.out.print(String.format(wordFormat, "\"" + dbOnlyWord + "\","));
+            if (++i % numOfWordsLine == 0) {
+                System.out.println(String.format(numFormat, i));
+            }
+        }
+        
+        System.out.println();
```

Comment:
As far as I see this code part is for debugging only. Please remove it or put under if(DEBUG) condition where DEBUG is false.
System.out.println should not be executed while running DBeaver in release build.

Reply:
Those main and filter methods at AltibaseSQLDialect.java is a kind of code generator to find Altibase only keywords and functions. As you recommended, I made filter method not to work when AltibaseConstants.DEBUG is false. 

Thanks for your advice :)